IFC 4.3/LinearPlacement_3/readme.md

```diff
@@ -0,0 +1,50 @@
+
+# LinearPlacement_3
+--- 
+
+### Intent
+
+This scenario demonstrates the usage of `IfcLinearPlacement` to setout road chevron signs by an alignment curve
+
+### Prerequisites
+
+This scenario builds upon following other scenarios:
+- UT_Align_1 alignment test data provided by APLITOP
+
+### Content
+
+This scenario builds upon following scenarios:
+- project structure
+  - `IfcProject`
+  - `IfcSite`
+  - `IfcRoad`
+- default unit assignment
+  - `IfcUnitAssignment`
+  - `IfcSIUnit`
+- alignment
+  - `IfcAlignment`
```

Comment:
I'd add at least a line here, sth in the ways of:
- for other alignment entities, see UT_Align_1 this example builds upon